pkg/selinux/setroubleshoot-view.jsx

```diff
@@ -361,16 +361,16 @@ export class SETroubleshootPage extends React.Component {
                 ];
                 // if the alert has level "red", it's critical
                 const criticalAlert = (itm.details && 'level' in itm.details && itm.details.level == "red")
-                    ? <ExclamationTriangleIcon className="ct-icon-exclamation-triangle pf-c-icon pf-m-lg" />
+                    ? <ExclamationTriangleIcon className="ct-icon-exclamation-triangle pf-v5-c-icon pf-m-lg" />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230516-073248-d32f52be-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)